getting-started/vs-code-integration/new-project-wizard.md

```diff
@@ -36,25 +38,49 @@ The **Create New Project** wizard provides two options to start you project:
     * **Dashboard**&mdash;A basic dashboard that showcases the Telerik UI TileLayout component and how individual blocks (tiles) can fetch data.
     * **Admin**&mdash;A small app that showcases a dashboard for administrators. The app uses some of the main Telerik UI components like the [Data Grid]({%slug grid-overview%}), [Chart]({%slug components/chart/overview%}), [TileLayout]({%slug tilelayout-overview%}), [Form]({%slug form-overview%}), and more.
 
-## Configuring the Project
+The available templates vary depending on the target framework and hosting model. All of the above templates are available for the Client and Server projects for .NET 6 and .NET 7. The Hybrid projects and the .NET 8 WebApp support only the **Start from Blank App** option.
+
+### Section 2: Select Project Name and Path
+
+This section allows you to type your project name and select the output folder for it.
+
+### Section 3: Select UI for Blazor Version
+
+This section allows you to select the preferred UI for Blazor version. The current latest version is pre-selected by default. Therefore, you must ensure that your system has access to the same version of the UI components - you can achieve this with the [Telerik NuGet feed](../../installation/nuget).
```

Comment:
Why is this using a relative link instead of a slug one? Let me know if I've missed some change to our linking strategy and need to know about it.

Also, did they make this a drop-down list/radio button list? Or do people still need to type a different version?

```suggestion
Select the preferred UI for Blazor version. Make sure that your system has access to the specified version of the UI components. You can achieve this with the [Telerik NuGet feed](../../installation/nuget).

The latest released version is pre-selected by default. 
```